proxy/src/test/java/com/wavefront/agent/preprocessor/AgentConfigurationTest.java

```diff
@@ -33,7 +33,7 @@ public void testLoadValidRules() {
     InputStream stream = PreprocessorRulesTest.class.getResourceAsStream("preprocessor_rules.yaml");
     config.loadFromStream(stream);
     Assert.assertEquals(0, config.totalInvalidRules);
-    Assert.assertEquals(60, config.totalValidRules);
+    Assert.assertEquals(61, config.totalValidRules);
```

Comment:
It seems like this test is failing. 
java.lang.AssertionError: 
Expected :61
Actual   :62
<Click to see difference>


	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:647)
	at org.junit.Assert.assertEquals(Assert.java:633)
	at com.wavefront.agent.preprocessor.AgentConfigurationTest.testLoadValidRules(AgentConfigurationTest.java:36)
	at